### DISCUSSION TITLE
Stuck when uploading a file

### DISCUSSION THREAD

**Discussion by rezuantpr** [filamentphp/filament#5694] [Help]
Can someone help me solve a problem with uploading files. When I try to upload a file, it gets stuck. I can't see the file in the "temp" directory and I cannot even cancel the upload. I have tried specifying all drives: local, public, s3. 

```
FileUpload::make('image')
                ->disk('local')
                ->directory('uploads')
```

<img width="546" alt="image" src="https://user-images.githubusercontent.com/28421025/219127550-e658c0ba-fb90-4fc3-97a1-b19d55310661.png">

I ran the "php artisan storage:link" command for public.
I specified APP_URL in the .env
And of course, I have a model with a _fillable nullable string_ field "image".

(but it doesn't matter, because I even cannot upload)

Any other read/write operations work fine.

I use localhost 

If this information is not detailed, I can create a test project reproducing this error

Filament version: v2.16

**Comment by danharrin** [comment]
Console errors?
